docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,16 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is kind of query known as a "non-scoring" query. It does not give a score,
+  it is only concerned about answering the question - "Does this document match?". 
+  The answer is always a simple, binary yes|no. This kind of query is said to be made 
```

Comment:
In the context of the glossary I would spell "yes|no" out and say `yes` or `no`.